geps/gep-713/index.md

```diff
@@ -1,1157 +1,1082 @@
 
-"Inherited Policy Attachment" is when a Policy object references a single object
-_and any child objects of that object_ (according to some defined hierarchy), and
-modifies fields of the child objects, or configuration associated with the child
-objects.
+### Policies and Policy Attachment
 
-In either case, a Policy may either affect an object by controlling the value
-of one of the existing _fields_ in the `spec` of an object, or it may add
-additional fields that are _not_ in the `spec` of the object.
+Often when applying a metaresource, the intent has to do with specifying *rules* that control the behavior of the targets in action. E.g., for networking resources, the augmented behavior specified by a metaresource targeting an object can relate to conditions to let traffic flow or to block traffic at particular points of the network; in a different scenario, it can describe rules to mutate the traffic while it flows through the network at the point represented by the target object.
 
-### Why use Policy Attachment at all?
+The semantics of *applying rules to a target* characterizes these particular instances of metaresources as *policies*, thus here emerging in relation to this pattern the concept of *Policy Attachment*.
 
+Policies are therefore a strict subset of metaresources which distinguish itself subtly from the whole on the fact that the augmented behavior described by policies resemble *conditional rules*, while the mechanism of targeting an object or set of objects (“attaching to objects”) is nonetheless exactly the same as in the general definition of metaresources.
 
-Consistent UX across GW implementations
+Without loss of generality, the very activation of the context (targets) of metaresources can be understood as the _conditions_ for the enforcement of the intended augment behavior (rules) that are specified in the metaresources, and therefore these can often be refered to as policies interchangeably.
 
-Support for common tooling such as gwctl that can compute and display effective policies at each layer
+## Applications
 
-Avoid annotation hell
+### Reasons for using metaresources and policies
 
+These are a few reasons for using metaresources and policies over another (possibly more direct) way to modify the spec (“augment the behavior”) of an object:
 
-### Direct Policy Attachment
+* Extending otherwise stable APIs – e.g. to specify additional network settings for the Kubernetes Service object.
+* Defining implementation-specific functionalities for otherwise common APIs－e.g. to specify implementation-specific behavior for Gateway API HTTPRoute objects.
+* Decoupling concerns for targeting personas with specific functionality and configuration－ delegation of responsibilities, fine-grained RBAC.
+* Decoupling responsibility over the management and implementation of the metaresources themselves.
+* Avoid alternatives based on annotations which are often non-standardized, poorly documented, and generally hard to maintain.
```

Comment:
Again, I would add that using policies helps maintain an expressive API because they try to self-document intent, unlike annotations/labels.